projects/lendai-us/design-system/src/lib/layout/sidenav-layout/sidenav-layout.stories.ts

```diff
@@ -67,7 +68,7 @@ export const SidenavLayout: Story = {
   render: props => ({
     props,
     template: `
-    <lai-sidenav-layout>
+    <lai-sidenav-layout [logoPath]="logoPath">
```

Comment:
there is no such input for `lai-sidenav-layout` as `logoPath`, where did it come from?